.github/workflows/dockerfilelint-else.yml

```
@@ -0,0 +1,15 @@
+name: dockerfile lint (else)
+
+on:
+  pull_request:
+    paths-ignore:
+      - ./Dockerfile
+
+jobs:
+  run_test:
+    name: test
```

```suggestion
  lint:
    name: Dockerfile lint
```
ジョブの名称が変なので揃えておきましょう